cmd/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hc"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
why is this version number getting bumped here?

Reply:
I just did that at one point to test that it was getting compliled will fix.